### PR TITLE
:bug:Source DV-360: Migrate from Bid Manager API v1.1 to v2

### LINKED ISSUE #23387: Source dv 360: Migrate from Bid Manager API v1.1 to v2
## Tell us about the problem you're trying to solve
Migrate to v2 to avoid service interruption, Bid Manager will support  v1.1 until April 27, 2023.

Please use the following useful resources to start migration:
• To find out the major updates in v2, see [V2 Migrate](https://developers.google.com/bid-manager/how-tos/v2-migration).

### PR DESCRIPTION
**Pull request by topefolorunso**
## What
Fixes #23387

## How
*Describe the solution*

## Recommended reading order
1. `source.py`
2. `streams.py`
3. `query_template.json`

## 🚨 User Impact 🚨
*Are there any breaking changes? What is the end result perceived by the user?*

*For connector PRs, use this section to explain which type of semantic versioning bump occurs as a result of the changes. Refer to our [Semantic Versioning for Connectors](https://docs.airbyte.com/contributing-to-airbyte/#semantic-versioning-for-connectors) guidelines for more information. **Breaking changes to connectors must be documented by an Airbyte engineer (PR author, or reviewer for community PRs) by using the [Breaking Change Release Playbook](https://docs.google.com/document/d/1VYQggHbL_PN0dDDu7rCyzBLGRtX-R3cpwXaY8QxEgzw/edit).***

*If there are breaking changes, please merge this PR with the 🚨🚨 emoji so changelog authors can further highlight this if needed.*

## Pre-merge Actions
*Expand the relevant checklist and delete the others.*

<details><summary><strong>Updating a connector</strong></summary>

### Community member or Airbyter

- Grant edit access to maintainers ([instructions](https://docs.github.com/en/github/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork#enabling-repository-maintainer-permissions-on-existing-pull-requests))
- Unit & integration tests added

### Airbyter

If this is a community PR, the Airbyte engineer reviewing this PR is responsible for the below items.

- Create a non-forked branch based on this PR and test the below items on it
- Build is successful
- If new credentials are required for use in CI, add them to GSM. [Instructions](https://docs.airbyte.io/connector-development#using-credentials-in-ci).

</details>

### CODE REVIEW

File: airbyte-integrations/connectors/source-dv-360/source_dv_360/queries/query_template.json [hunk 1]
```diff
@@ -1,13 +1,21 @@
 {
-  "kind": "doubleclickbidmanager#query",
   "queryId": "0",
   "metadata": {
     "title": "",
-    "dataRange": "CUSTOM_DATES",
+    "dataRange": {
```
Comment (lazebnyi) [inline airbyte-integrations/connectors/source-dv-360/source_dv_360/queries/query_template.json:5]: is it braking changes?

**Review by lazebnyi — COMMENTED** [review]
need update version and change log

File: airbyte-integrations/connectors/source-dv-360/source_dv_360/queries/query_template.json (hunk 1 above)
Comment (topefolorunso) [reply airbyte-integrations/connectors/source-dv-360/source_dv_360/queries/query_template.json:5]: No, not breaking change.

**Comment by topefolorunso** [comment]
> need update version and change log

Done @lazebnyi

**Review by lazebnyi — APPROVED** [review]

**Comment by andremacola** [comment]
Is anyone looking at this pull request?

**Comment by marcosmarxm** [comment]
This connector was archived and will be deprecated. More about the notice you can read here: https://docs.airbyte.com/integrations/sources/dv-360#deprecation-notice

**Pull request closed without merging**